experiments/AtmosLES/taylor-green.jl

```diff
@@ -0,0 +1,219 @@
+using CLIMA.ODESolvers
+# - Required for utility of spatial filtering functions (e.g. positivity preservation)
+using CLIMA.Mesh.Filters
+# - Required so functions for computation of moist thermodynamic quantities is enabled.
+using CLIMA.MoistThermodynamics
+# - Required so we may access our variable arrays by a sensible naming convention rather than by numerical array indices.
+using CLIMA.VariableTemplates
+# - Required so we may access planet parameters ([CLIMAParameters](https://climate-machine.github.io/CLIMAParameters.jl/latest/) specific to this problem include the gas constant, specific heats, mean-sea-level pressure, gravity and the Smagorinsky coefficient)
+using CLIMAParameters
+using CLIMAParameters.Atmos.SubgridScale: C_smag
+using CLIMAParameters.Planet: R_d, cp_d, cv_d, MSLP, grav
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+function init_greenvortex!(bl, state, aux, (x, y, z), t)
+    # Problem float-type
+    FT = eltype(state)
+
+    # Unpack constant parameters
+    R_gas::FT = R_d(bl.param_set)
+    c_p::FT = cp_d(bl.param_set)
+    c_v::FT = cv_d(bl.param_set)
+    p0::FT = MSLP(bl.param_set)
+    _grav::FT = grav(bl.param_set)
+    γ::FT = c_p / c_v
+
+
+
+    # Compute perturbed thermodynamic state:
+    ρ = FT(1.178)      # density
+    ρu = SVector(FT(0), FT(0), FT(0))                   # momentum
+    #State (prognostic) variable assignment
+    e_pot = FT(0)# potential energy
+    Pinf = 101325
+    Uzero = FT(100)
+    p = Pinf + (ρ * Uzero / 16) * (2 + cos(z)) * (cos(x) + cos(y)) 
+    u = Uzero * sin(x) * cos(y) * cos(z)
+    v = - Uzero * cos(x) * sin(y) * cos(z)
+    e_kin = 0.5 * (u^2+v^2)
+    T = p / (ρ * R_gas)
+    e_int = internal_energy(bl.param_set, T, PhasePartition(FT(0)))
+    ρe_tot = ρ * (e_kin + e_pot + e_int)
+    # Assign State Variables
+    state.ρ = ρ
+    state.ρu = SVector(FT(ρ * u),FT(ρ * v),FT(0))
+    state.ρe = ρe_tot
+end
+
+# ## [Model Configuration](@id config-helper)
+# We define a configuration function to assist in prescribing the physical model. The purpose of this is to populate the [`CLIMA.AtmosLESConfiguration`](@ref LESConfig) with arguments appropriate to the problem being considered.
+"""
+    config_risingbubble(FT, N, resolution, xmax, ymax, zmax)
+
+Arguments
+- FT = Floating-point type. Currently `Float64` or `Float32`
+- N  = DG Polynomial order
+- resolution = 3-component Tuple (Δx, Δy, Δz) with effective resolutions for Cartesian directions
+- xmax, ymax, zmax = Domain maximum extents. Assumes (0,0,0) to be the domain minimum extents unless otherwise specified.
+
+Returns
+- `config` = Object using the constructor for the `AtmosLESConfiguration`
+"""
+function config_greenvortex(FT, N, resolution, xmax, ymax, zmax,xmin,ymin,zmin)
+
+    ode_solver = CLIMA.ExplicitSolverType(
+        solver_method = LSRK144NiegemannDiehlBusch,
+    )
+
+    # Since we want four tracers, we specify this and include
```

Comment:
Think this can be removed? I don't see any tracers being added/incorporated in the model.

Reply:
> Think this can be removed? I don't see any tracers being added/incorporated in the model.

Yes, to be cleaned up. This driver was built from an existing one and was never cleaned.